UM2N/generator/unstructured_mesh.py

```diff
@@ -0,0 +1,194 @@
+"""
+Module for handling generating unstructured meshes.
+"""
+
+from firedrake.mesh import Mesh
+
+import abc
+import gmsh
+import numpy as np
+import os
+import random
```

Comment:
When I set up the linting I started added exclusions to avoid the linter changing the entire codebase. Plan to add them back in as we go. Fixed in 4e48f6bcf1a1a02b1ec6685a16e402f02f37a059.

Reply:
Makes sense :) thanks